libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Memory/DialogStateManager.cs

```diff
@@ -147,7 +147,7 @@ public virtual MemoryScope ResolveMemoryScope(string path, out string remainingP
             }
 
             remainingPath = string.Empty;
-            return GetMemoryScope(scope) ?? throw new ArgumentOutOfRangeException(GetBadScopeMessage(path));
```

Comment:
>throw new ArgumentOutOfRangeException(GetBadScopeMessa [](start = 44, length = 54)

Hmm.  I kind of like getting rid of this, but when I proposed before you felt it was good to keep...